easybuild/easyconfigs/s/SHAPEIT4/SHAPEIT4_4.2.0_use_EB_settings.patch

```diff
@@ -24,18 +21,16 @@ Adam Huffman
  
  #HTSLIB LIBRARY [SPECIFY YOUR OWN PATHS]
  #HTSLIB_INC=/software/UHTS/Analysis/samtools/1.4/include
-@@ -25,12 +25,12 @@
- #CXXFLAG=-O3 -march=native
+@@ -22,10 +22,10 @@
+ #COMPILER & LINKER FLAGS
+ 
+ #Best performance is achieved with this. Use it if running on the same plateform you're compiling.
+-#CXXFLAG=-O3 -march=native
++CXXFLAG=-O3 -march=native
```

Comment:
Shouldn't we rather set it to whatever the CXXFLAGS environment variable is?

Reply:
@Micket would the same apply to `LDFLAG`?

Reply:
Sure, probably wouldn't hurt (though it seems the compiler did pick up the library paths anyway via one of the environment variables since it worked).

Reply:
Yes, I agree, just do this via `buildopts`:

```python
buildopts = 'CXX="$CXX" CXXFLAG="$CXXFLAGS" LDFLAG="$CXXFLAGS"'

Reply:
Now done. I had to fix the original `toolchainopts`, which caused an error after using `CXXFLAGS`.